### DISCUSSION TITLE
Better divide and spaceing support

### DISCUSSION THREAD

**Discussion by quick007** [tailwindlabs/tailwindcss#4564] [Ideas]
Currently, if you do `divide-y space-y-4` it looks like this
![image](https://user-images.githubusercontent.com/69743171/120877996-69617080-c56e-11eb-8073-77851381fa1e.png)
I believe it should put the divider in the middle, like this (recreated using margin)
![image](https://user-images.githubusercontent.com/69743171/120878020-901fa700-c56e-11eb-9744-75f1f4755ff6.png)

**Comment by jenstornell** [comment]
This feature feels a bit like a workaround from the time when `gap` was lacking browser support.

Now when gap is supported in all major browsers I rekommend you to use `flex flex-col gap-y-4` instead.

https://caniuse.com/?search=gap
